tags.json

```
@@ -59,6 +59,7 @@
   "credit-card": ["purchase", "payment", "cc"],
   "crop": ["photo", "image"],
   "crosshair": ["aim", "target"],
+  "crown": ["crown"],
```

Add a name as a tag is not necessary, search engine also search by name.
Here some other suggestions as tags

```suggestion
  "crown": ["king", "winner", "favourite"],
```